diff --git a/node_modules/@greenwood/cli/src/plugins/resource/plugin-standard-html.js b/node_modules/@greenwood/cli/src/plugins/resource/plugin-standard-html.js
index 8f06225..0328e32 100644
--- a/node_modules/@greenwood/cli/src/plugins/resource/plugin-standard-html.js
+++ b/node_modules/@greenwood/cli/src/plugins/resource/plugin-standard-html.js
@@ -105,7 +105,25 @@ class StandardHtmlResource extends ResourceInterface {
     }
 
     if (matchingRoute.isSSR) {
-      const routeModuleLocationUrl = new URL(`./${matchingRoute.filename}`, pagesDir);
+      console.log({ matchingRoute });
+      // {
+      //   matchingRoute: {
+      //     data: {},
+      //     filename: 'blissfest-2023.js',
+      //     id: 'blissfest-2023',
+      //     label: 'Blissfest 2023',
+      //     imports: [],
+      //     resources: [],
+      //     outputPath: '/photos/blissfest-2023/index.html',
+      //     path: '/photos/blissfest-2023/',
+      //     route: '/photos/blissfest-2023/',
+      //     template: 'page',
+      //     title: null,
+      //     isSSR: true,
+      //     prerender: false
+      //   }
+      // }
+      const routeModuleLocationUrl = new URL(`./${matchingRoute.route.replace(`${matchingRoute.id}/`, `${matchingRoute.id}.js`)}`, pagesDir);
       const routeWorkerUrl = this.compilation.config.plugins.find(plugin => plugin.type === 'renderer').provider().executeModuleUrl;
 
       await new Promise(async (resolve, reject) => {
